### PR TITLE
Tested

### PR DESCRIPTION
**Pull request by CooviM**
Deleted libraries, thermostat codes, rearranged files and added python set up files

### CODE REVIEW

File: amigos/amigos/log.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Dict([(u'Datetime', datetime.datetime(2019, 6, 14, 14, 1, 53)), (u'RecNbr', 179), (u'R6', 1086.823486328125), (u'TCDT', nan), (u'R10', 1086.2347412109375), (u'R2_5', 1085.1212158203125), (u'R6_5', 1086.333740234375), (u'R8_5', 1086.3167724609375), (u'Q', nan), (u'R40', 1085.6849365234375), (u'Batt_volt', 11.579564094543457), (u'R4_5', 1085.0435791015625), (u'Ptemp_C', 24.85491943359375), (u'DT', nan), (u'R20', 1085.9552001953125)])
```
Comment (wallinb) [inline amigos/amigos/log.txt]: Included by accident?

File: amigos/amigos/gpio.py [hunk 2]
```diff
@@ -136,8 +131,69 @@ def power_up(bit):
...
+        __toggle(bit-1)
+        subprocess.call(
+            "echo {0} > /sys/class/gpio/pwr_ctl/data".format(hex(int(bit_str, 2))), shell=True)
+        print("ok")
+        __update_bit(bit_string[0] + ','+bit_str)
+
+
+def CR1000_off(bit):
+    """
+    Turn the power off gps module on after toggling the bit
```
Comment (wallinb) [inline amigos/amigos/gpio.py]: Watch out for innacurate comments/docstrings

File: amigos/amigos/cr100x_test.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import datetime
+from pycampbellcr1000 import CR1000
+device = CR1000.from_url('tcp:192.168.1.30:6785')
+data = device.get_data('Public')
```
Comment (wallinb) [inline amigos/amigos/cr100x_test.py]: Well that was easy!

File: amigos/amigos/Weather_Reading.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
+#Sid Arora
```
Comment (wallinb) [inline amigos/amigos/Weather_Reading.py]: We should modularize this and integrated with cli if not done already.

Maybe need a separate folder for demo-type scripts like this?

There is merge conflict annotation here still as well.

File: amigos/amigos/cr100x_test.py (hunk 3 above)
Comment (CooviM) [reply amigos/amigos/cr100x_test.py]: yep. the data come as a dict. Thus formating is the only thing to do here

File: amigos/amigos/log.txt (hunk 1 above)
Comment (CooviM) [reply amigos/amigos/log.txt]: haha! It has already been deleted from my file system.

File: amigos/amigos/Weather_Reading.py (hunk 4 above)
Comment (CooviM) [reply amigos/amigos/Weather_Reading.py]: Undertood. I will let ssid know. About the merge problem, I will fix that tomorrow.

File: amigos/amigos/gpio.py (hunk 2 above)
Comment (CooviM) [reply amigos/amigos/gpio.py]: That what copy-paste does

File: amigos/amigos/gpio.py [hunk 5]
```diff
@@ -136,8 +131,69 @@ def power_up(bit):
...
         subprocess.call("echo 0xFF > /sys/class/gpio/pwr_ctl/data", shell=True)
-        print("Power is up on peripherals")
-    __update_bit('0b11111111,1b11111111')
+        print("ok")
+        __update_bit('0b11111111,0b11111111')
+
+
+def CR1000_on(bit):
+    """
+    Turn the power off gps module on after toggling the bit
```
Comment (CooviM) [inline amigos/amigos/gpio.py]: This section was copied and paste. So will be fixed by the appropriate person tomorrow

**Comment by wallinb** [comment]
Rolled into PR #19

**Pull request closed without merging**